### DISCUSSION TITLE
wire:navigate is lower than inertia routing

### DISCUSSION THREAD

**Discussion by nlvedwin** [livewire/livewire#5863] [4. General]
Hi @calebporzio , 

First of all, I'm very excited about this feature! Great job on livewire v3!

I'm doing a test on the wire navigate and discovered that inertia spa routing has a better response time. Is there something wrong in my local that could cause the problem?

wire:navigate
![image](https://github.com/livewire/livewire/assets/78135357/f5c53ac5-1e88-4a75-a768-7b0961cc4eba)

inertia
<img width="325" alt="image" src="https://github.com/livewire/livewire/assets/78135357/0f88a15d-c6c7-4106-9cbb-92b5a7771a01">

**Comment by joshhanley** [comment]
@nlvedwin I don't think there would be anything wrong on your local machine. It's likely slower as Livewire renders the view on the backend where as inertia wouldn't.

Caleb will be releasing a tool soon that will allow you to investigate the timing better, so I'd say wait for that and test it with that once it has been released.

Hope this helps!

**Reply by nlvedwin** [reply]
thanks for responding @joshhanley, when i tested it again it's now faster i think it's just my computer

**Reply by joshhanley** [reply]
@nlvedwin no worries! 🙂
